taxonomies/food/ingredients.txt

```diff
@@ -38460,37 +38421,27 @@ wikipedia:en: https://en.wikipedia.org/wiki/Juice
 # 1687 products in 6 languages @2018-10-04
 
 # don't put under fruit juice, because concentrate fruit juices that are not reconstituted to 100% of their initial volume don't count for the nutriscore
-en: fruit juice concentrate
+< en:fruit juice concentrate
+en: fruit juice concentrate, fruit juice from concentrate, fruit juices from concentrate
 da: Frugtjuicekoncentrat
-de: Fruchtsaftkonzentrat, Fruchtsäfte Konzentrat
-es: Jugo de frutas concentrado
-fi: hedelmämehutiiviste
-fr: jus concentré de fruits, jus de fruits concentrés
+de: Fruchtsaftkonzentrat, Fruchtsäfte Konzentrat, Fruchtsaft auf Konzentrat Basis, Fruchtsaft aus Fruchtsaftkonzentrat, Fruchtsaft aus Fruchtsaftkonzentraten, Fruchtsäfte aus Konzentraten
+es: Jugo de frutas concentrado, zumo de frutas a partir de concentrado, jugo de frutas procedente de concentrado
+fi: hedelmämehutiiviste, hedelmämehu tiivisteestä, hedelmämehut tiivisteestä, tiivisteestä valmistetut täysmehut
+fr: jus concentré de fruits, jus de fruits concentrés, jus de fruits à base de concentré, jus de fruits à base de concentrés, jus de fruits à base de jus concentrés
 he: תרכיז מיץ פרות, תרכיז מיץ פירות, תרכיז מיצי פירות
-hr: koncentrirani voćni sok
-it: Concentrato di succo di frutta
-nl: vruchtensapconcentraat
-pl: Koncentrat soku owocowego, zagęszczone soki owocowe
+hr: koncentrirani voćni sok, vočni sok iz koncentrata, voćnog soka od koncentriranog soka, soka više vrsta voća od koncentriranog soka
+it: Concentrato di succo di frutta, succhi di frutta a base di concentrato, succhi di frutta a base di concentrati
+nl: vruchtensapconcentraat, vruchtsap van concentraat, vruchtensap uit concentraat
+pl: Koncentrat soku owocowego, zagęszczone soki owocowe, sok owocowy z koncentratu, soki owocowe z koncentratu, sok owocowy z koncentratu z, soki z soków zagęszczonych, soki z zagęszczonych soków, soki owocowe z zagęszczonych soków owocowych
 ru: Концентрат фруктового сока
-sv: fruktjuicekoncentrat
+sv: fruktjuicekoncentrat, fruktjuice från koncentrat, juicer framställd av koncentrat, juice från koncentrat, frukt juice från koncentrat
 nova:en: 4
+# openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:jus-de-fruits-à-base-de-concentré
+# 650 products in 7 languages @2018-10-04
```

Comment:
This one should have not been merge… I have to look into it, it might be a bug of taxonomy-editor (that removes stop words when matching entries)